packages/main/src/Table.hbs

```diff
@@ -35,7 +35,7 @@
 <div id="after" role="none" tabindex="0" ui5-table-dummy-focus-area></div>
 
 {{#*inline "growingRow"}}
-    <ui5-table-row id="growing-row">
+    <ui5-table-row id="growing-row" ui5-growing-row>
```

Comment:
what is the reason for that?
we can identify it from its id and it is single for the table.

Reply:
Same reasoning, why I've added ui5-table-row-base.

Applications could go ahead an set a random row with id="growing-row" and then, it is not possible to discern our internal growing row from the external one (e.g. in a focusin, keyup event handler)

Very unlikely scenario, but just to be on the safe side and have no ID restrictions. Of course this issue could also occur, if the app is setting the ui5-growing-row attribute, but then I think it is quite malicious from their side.

Reply:
for rows I understand but not for the table
what I mean is instead of checking the attribute we can check ` === table._growingRow``
but ok